### PR TITLE
Fix display for profile & group & planning if plugin not activated

### PR DESCRIPTION
**Pull request by tsmr**
Fix display for profile & group
Fix planning if plugin not activated

### CODE REVIEW

File: inc/group.class.php [hunk 1]
```diff
@@ -246,8 +246,8 @@ function showForm($ID, $options=array()) {
       echo "<td>";
       Html::autocompletionTextField($this, "name");
       echo "</td>";
-      echo "<td rowspan='9' class='middle'>".__('Comments')."</td>";
-      echo "<td class='middle' rowspan='9'>";
+      echo "<td rowspan='10' class='middle'>".__('Comments')."</td>";
```
Comment (orthagh) [inline inc/group.class.php]: I don't understant this part, when Group::showForm are displayed in planning ?

File: inc/planning.class.php [hunk 2]
```diff
@@ -1126,6 +1126,9 @@ static function showSingleLinePlanningFilter($filter_key, $filter_data, $options
          $group->getFromDB($actor[1]);
          $title = $group->getName();
       } else if($filter_data['type'] == 'event_filter') {
+         if (!($item = getItemForItemtype($filter_key))) {
+            return false;
+         }
```
Comment (orthagh) [inline inc/planning.class.php]: Maybe, you can compact this test with the next one (with a || operator)

File: inc/profile.class.php [hunk 3]
```diff
@@ -574,7 +574,7 @@ function showForm($ID, $options=array()) {
 
       $onfocus = "";
       $new     = false;
-      $rowspan = 5;
+      $rowspan = 4;
```
Comment (orthagh) [inline inc/profile.class.php]: same as Group::showForm, could you explain a little ?

**Comment by orthagh** [comment]
Oh also, it's not mandatory for this pr, but you should consider using branch instead of your master when pushing PR

File: inc/profile.class.php (hunk 3 above)
Comment (tsmr) [inline inc/profile.class.php]: No it's nor related to planning to, it's only to fix display with forget rowspan

**Comment by orthagh** [comment]
@tsmr I was in pain to understand the relation between parts of this pr.
In fact, you have two fixes for this pr and the issue #1109 
It's bad.

Please, open separated issues who clearly describe the bug and do separate pull requests.
As said in my previous comment (https://github.com/glpi-project/glpi/pull/1108#issuecomment-251666870), you could use branches to do that.
Here is an simple example:
```bash
# change your git repository to master branch and retrieve last changes
git checkout master
git pull origin master

# create a new branch (from master)
git checkout -b my_branch

# do your changes and commit
git add inc/....
git commit

# push on your own fork (you need a corresponding remote)
git push tsmr

# move on master
git checkout master
```

On a opened pr, you could add more commits to your branch and push them to your fork.

**Comment by tsmr** [comment]
I understood. I make branches for new PR & separate goal of them.

PS : I continue to think that do PR for only change one rowspan add charge for a little change :(

**Comment by orthagh** [comment]
Ok, great for separate goal/pr.

I aggre it can be annoying for a small fix/change to do this process, but systematic reviews avoid regressions. We are in a process of enhance quality by workflow, test, etc.

With some git practice, you'll not lose time in the future. :)

BTW, i merge this one as the change are ok.

**Pull request merged**